### DISCUSSION TITLE
Can I use a path in NPM?

### DISCUSSION THREAD

**Discussion by Timmiej93** [NginxProxyManager/nginx-proxy-manager#1288] [Q&A]
I'm using DuckDNS, and I want to have multiple services under one domain. Obviously (at least I think so), `service1.myname.duckdns.org` isn't going to work, so I need something like `myname.duckdns.org/service1`. I _thought_ this could be done with custom locations in NPM, but the more I struggle with it, the more I start to doubt that.

Put simply: Can I have `myname.duckdns.org` pointing to one thing (say my web page), and `myname,duckdns.org/homeassistant` pointing to something else, like HA? If so, how do I do that?

I tried all kinds of things in the custom locations section of my proxy hosts. `/homeassistant`, `myname,duckdns.org/homeassistant`, etc., etc., nothing seems to work. Am I misunderstanding what custom locations are, or is something else going on?

**Comment by papatistos** [comment]
You're not alone. See here, for example: https://www.reddit.com/r/selfhosted/comments/lqgopt/nginx_proxy_manager_admin_site_behind_nginx/

**Reply by mcc311** [reply]
Same problem here

**Comment by Zonsopkomst** [comment]
I'm trying to do the same thing- either subdomains or subfolders using DuckDNS.  It at least appears that some people are able to do this with their home servers (search reddit discussions for example), and they seem to be doing the same things we are trying, but there is very little documentation in my experience.  I can set up proxy hosts or redirection hosts and even receive Let's Encrypt certs for sub1.mydomain.duckdns.org and the like, however when I go to the hyperlink in a browser, it remains unsecured.  Would be much appreciated if someone could walk us through the correct way of setting up subdomains or subfolders in NPM for use with DuckDNS.  Thanks!

BTW- my main mydomain.duckdns.org is SSL secure.  That's why it is strange and frustrating that I can't get the subdomains or subfolders to do the same using a wildcard cert like `*.mydomain.duckdns.org `or `mydomain.duckdns.org/*` or even individually as `sub1.mydomain.duckdns.org` or `mydomain.duckdns.org/sub1`.

**Comment by mcc311** [comment]
Is there anyone who can solve this problem now?
